app/src/main/res/drawable/bg_brush_color_red.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Maybe we should have one shape and set color dynamically to accommodate having many colors

Reply:
@shubertm That's a good suggestion.
I've updated it here: 15bde026dd36fbf8d7e6391f54450da6aad34540